spec/unit/service_spec.rb

```diff
@@ -399,6 +399,15 @@ module Backend
           expect(Service.default_id(service.provider_key)).to eq '7002'
         end
 
+        it 'deletes all service errors' do
+          Service.save! id: 7002, provider_key: 'foo', default_service: true
+          ErrorStorage.store(service.id, ApplicationNotFound.new('foo'))
```

Comment:
I would store at least 2 to make sure that this works correctly and does not delete just one.